src/xrpld/ledger/ApplyViewImpl.h

```diff
@@ -86,6 +122,8 @@ class ApplyViewImpl final : public detail::ApplyViewBase
 
 private:
     std::optional<STAmount> deliver_;
+    std::vector<STObject> batchExecution_;
+    std::optional<STObject> batchPrev_;
```

Comment:
might be worth renaming this to make it clear that it stores account root's previous fields. something like `batchPrevAcctFields_`